### PR TITLE
Big modules update

### PR DESCRIPTION
**Pull request by jasmezz**
This updates a bunch of modules (see changelog). For some (e.g. RGI and Pyrodigal) the logic in their (sub)workflows had to be adjusted. While I was going through the workflows, I adjusted the formatting where necessary.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,singularity --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/rgi/cardannotation/main.nf [hunk 1]
```diff
@@ -0,0 +1,61 @@
+process RGI_CARDANNOTATION {
```
Comment (jfy133) [inline modules/nf-core/rgi/cardannotation/main.nf:1]: Do we run this module?

File: nextflow.config_old [hunk 2]
```diff
@@ -0,0 +1,425 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (jfy133) [inline nextflow.config_old:2]: TO be removed? What is this exactly?

File: workflows/funcscan.nf [hunk 3]
```diff
@@ -240,29 +247,31 @@ workflow FUNCSCAN {
...
     //
-    ch_multiqc_config                     = Channel.fromPath("$projectDir/assets/multiqc_config.yml", checkIfExists: true)
-    ch_multiqc_custom_config              = params.multiqc_config ? Channel.fromPath(params.multiqc_config, checkIfExists: true) : Channel.empty()
-    ch_multiqc_logo                       = params.multiqc_logo ? Channel.fromPath(params.multiqc_logo, checkIfExists: true) : Channel.empty()
-    summary_params                        = paramsSummaryMap(workflow, parameters_schema: "nextflow_schema.json")
-    ch_workflow_summary                   = Channel.value(paramsSummaryMultiqc(summary_params))
+    ch_multiqc_config                     = Channel.fromPath( "$projectDir/assets/multiqc_config.yml", checkIfExists: true )
+    ch_multiqc_custom_config              = params.multiqc_config ? Channel.fromPath( params.multiqc_config, checkIfExists: true ) : Channel.empty()
+    ch_multiqc_logo                       = params.multiqc_logo ? Channel.fromPath( params.multiqc_logo, checkIfExists: true ) : Channel.empty()
+    summary_params                        = paramsSummaryMap( workflow, parameters_schema: "nextflow_schema.json" )
```
Comment (jfy133) [inline workflows/funcscan.nf:265]: Might result in template conflicts but OK for now

**Review by jfy133 — COMMENTED** [review]
Please add hAMRonization biorxiv preprint to CITATIONS.md and the auto-methods text 

- CHANGELOG: missing tabix version bump
- MODULES.CONF: Missing RGI CARDANNODATION entry
- OUTPUT.MD: Missing RGI CARDANNODATION entry

File: workflows/funcscan.nf (hunk 3 above)
Comment (jasmezz) [reply workflows/funcscan.nf:265]: Template linting is fine apparently :woman_shrugging:

File: nextflow.config_old (hunk 2 above)
Comment (jasmezz) [reply nextflow.config_old:2]: Oops, leftover of my debugging. Will remove.

**Comment by jfy133** [comment]
- OUTPUT.MD: Missing RGI CARDANNODATION entry?
- And for citations does CARD need ot go into https://github.com/nf-core/funcscan/blob/modules-update/subworkflows/local/utils_nfcore_funcscan_pipeline/main.nf#L196? Or is it same as rGI citation?

**Review by Darcy220606 — APPROVED** [review]
Superb!

**Comment by jasmezz** [comment]
It's the same citation as RGI which is updated upon every CARD release (according to [CARD website](https://card.mcmaster.ca) and [RGI repo](https://github.com/arpcard/rgi)). So thanks for checking, I will update the 2020 version with 2023 :+1:

File: CHANGELOG.md
[the remark this answers is not in the document]
Comment (jasmezz) [reply CHANGELOG.md:1]: All checked again now :heavy_check_mark:

**Comment by jasmezz** [comment]
@nf-core-bot fix linting

**Review by jfy133 — APPROVED** [review]

**Pull request merged**